electrum/gui/qml/qewallet.py

```diff
@@ -158,16 +158,17 @@ def on_event_request_status(self, wallet, key, status):
                 self.historyModel.init_model()
 
     @event_listener
-    def on_event_invoice_status(self, wallet, key):
+    def on_event_invoice_status(self, wallet, key, status):
```

Comment:
`qeinvoicelistmodel.py` also needs updating (?)
https://github.com/spesmilo/electrum/blob/eb00012c951e10855841d9e9daa19125cb3c04a2/electrum/gui/qml/qeinvoicelistmodel.py#L143

Reply:
Yes, the listener in `qeinvoicemodel.py` was added after I made this PR so it needs to be included here